utils/testutil/assert.go

```
@@ -0,0 +1,21 @@
+package testutil
+
+import (
+	"testing"
+)
+
+// Should checks error and if present, fails the test case.
+// Example usage: testutil.Should(t, task.Create(&r))
+func Should(t *testing.T, err error) {
+	if err != nil {
+		t.Errorf(err.Error())
```

golangci-lint (2.12.2)

[error] SA1006: printf-style function with dynamic format string and no further arguments should use print-style function instead

(staticcheck)